v2/cli/init_templates.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for Brigade projects
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  ## Subscribe below to any events that should trigger your script.
+  ## The commented example depicts a subscription to "exec"
+  ## events originating from the Brigade CLI. 
+  # eventSubscriptions:
+  # - source: brigade.sh/cli
+  #   types:
+  #   - exec
+  workerTemplate:
+    # logLevel: DEBUG
+    {{- if eq .GitCloneURL "" }}
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{- nindent 8 .Script }}
+    {{- else }}
+    git:
+      cloneURL: {{ .GitCloneURL }}
```

Comment:
Under `cloneURL`, let's add the following:

```
ref: refs/heads/master
```

Especially given that more and more people are using `main` instead of `master`, this will make it much more obvious how to change this if desired.